pymatgen/io/aims/parsers.py

```diff
@@ -58,6 +58,12 @@ def __init__(self, message: str) -> None:
 }
 
 
+def voigt_to_full_stress_conv(voigt_stress: Sequence[float]) -> Matrix3D:
+    """Transform voigt vector to a full 3x3 matrix."""
+    xx, yy, zz, yz, xz, xy = np.array(voigt_stress).flatten()
+    return np.array([[xx, xy, xz], [xy, yy, yz], [xz, yz, zz]])
```

Comment:
Can't we use `ase.stress.voigt_6_to_full_3x3_stress` instead of defining our own?

Reply:
We can I was not sure the recommendation was for the use of ASE functions in non-ASE parts since it is an optional dependency

Reply:
You were already importing from `ase.stress` but I just realized that was in a test file. You're right, in a non-test file, we have to wrap in `try: import ase except ImportError: ...`.

Better still though, we already have the [`Tensor` class for this](https://github.com/tpurcell90/pymatgen/blob/db52f3bf37224cfff2c8df363ad151059196f338/pymatgen/core/tensors.py#L40). It has a `from_voigt` method.

Reply:
I converted this to use the Tensor object instead (sorry for the delay I got my new personal computer so can now actually work again)

Reply:
Thanks! 👍